### ISSUE TITLE
OpenWeather API unauthorised error

### ISSUE THREAD

**Issue by Gravemind2015** [breezy-weather/breezy-weather#234]
### Steps to reproduce

Add a fresh free personal openweather api key into Settings>Weather Provider>OpenWeather>API key. Leave onecall version on 2.5 (happens on 3 as well).

### Expected behavior

API key should work and weather should be fetched.

### Actual behavior

App reports API access unauthorised

### Weather source used

OpenWeather 

### Breezy Weather version

4.4.0-beta

### Android version

Android 12 (Stock Samsung OneUI 4.0)

### Device

Samsung Galaxy M51

### Other details

I used to use a key with my tasker tasks. I can verify (via tasker) that new keys generated in openweather account do indeed work. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I read the [Help](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) documentation.
- [X] I have updated the app to **[latest version](https://github.com/breezy-weather/breezy-weather/releases/latest)**.

**Comment by Gravemind2015** [comment]
I investigated a little more. There are error notifications from breeze which when tapped, open a pdf file that basically sayas that API limit has reached.

Curiously, tasker is somehow still able to fetch data, but even using that particular API key doens't work in breeze. Also tested geometric weather but it also errors out. 

Maybe this is something broken in my openweather account rather than this app. Apologies for unnecessary bug report. I'll be closing this for now.

**Comment by papjul** [comment]
If you just created a key, you need two things:
- Use v3.0
- Wait a moment (it seems the key you just created doesn't work straight away)

**Comment by Gravemind2015** [comment]
Here's the thing: all keys I tested, I gave them about 10 hours after creation and they do work (checked via tasker). For some reason both breezy and geometric weather didn't work with them. I even tried adding an old key that I use in tasker, that didn't work either. 

But then I requested an api key from a friend and used that, and breezy worked just fine. This leads me to believe there's something stupid on my account/setup somewhere, but I cannot be sure because tasker works shamelessly with all keys. 😁
